sway/tree/layout.c

```diff
@@ -123,12 +123,11 @@ struct sway_container *container_remove_child(struct sway_container *child) {
 
 enum sway_container_layout container_get_default_layout(
 		struct sway_container *output) {
-	/* TODO WLR
 	if (config->default_layout != L_NONE) {
-		//return config->default_layout;
+		return config->default_layout;
 	} else if (config->default_orientation != L_NONE) {
 		return config->default_orientation;
-	} else */if (output->width >= output->height) {
+	} else if (output->width >= output->height) {
```

Comment:
Can't get this to work. Tried to resize the output so that height > width with `default_orientation auto`, it still opens new views on the right.

Reply:
is it supposed to update the orientation of the workspace when this becomes true on layout/mode changes?